test/e2e/app/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.21
 
 require (
 	github.com/cometbft/cometbft v0.38.0
+	github.com/cometbft/cometbft/api v1.0.0
```

Comment:
I don't yet know how to control this version for a standalone `./api` module.

Reply:
See https://go.dev/doc/modules/managing-source#multiple-module-source

Reply:
So at some point in preparing the release, we need to push an `api/v1.0.0-alpha.1` tag and update the require lines in `go.mod` files, correct?

Reply:
I think that right now, with the `replace` tag in `go.mod`, we can use whatever version number we want. I think the convention's to use `v0.0.0` until you're ready to actually tag a version of the submodule.

But yes, we'll eventually need to tag `api/v1.0.0-alpha.1` in order to publish the submodule.

Reply:
OK, I've changed the required version to `v0.0.0` as one of the squashed changes in https://github.com/cometbft/cometbft/pull/1671/commits/dc5de27f38373ed91bd4210c13c2929abb322fca